packages/cardano-services/src/Program/programs/types.ts

```diff
@@ -39,7 +40,8 @@ export enum ProviderServerOptionDescriptions {
   DisableDbCache = 'Disable DB cache',
   DisableStakePoolMetricApy = 'Omit this metric for improved query performance',
   EpochPollInterval = 'Epoch poll interval',
-  TokenMetadataCacheTtl = 'Token Metadata API cache TTL in minutes',
+  AssetCacheTtl = 'Asset info and NFT Metadata cache TTL in seconds (600 by default)',
+  TokenMetadataCacheTtl = 'Token Metadata API cache TTL in seconds',
```

Comment:
TODO: check how it's being set - this had wrong description (was being used as seconds)

Reply:
Lace backend is using the default